Description: Initial GNU/kFreeBSD support for openjdk-7 (Corba part)
 - Alter build system to consider GNU/kFreeBSD like linux
   since this port is libc based.
   openjdk/corba/make/common/shared/Platform.gmk
Author: Damien Raude-Morvan <drazzib@debian.org>
Last-Update: 2011-06-28
Forwarded: no
Index: openjdk/corba/make/common/shared/Platform.gmk
===================================================================
--- openjdk/corba/make/common/shared/Platform.gmk.orig	2011-07-16 01:48:27.000000000 +0200
+++ openjdk/corba/make/common/shared/Platform.gmk	2011-07-16 01:49:37.000000000 +0200
@@ -147,7 +147,7 @@
 endif
 
 # Platform settings specific to Linux
-ifeq ($(SYSTEM_UNAME), Linux)
+ifneq (,$(filter Linux GNU/kFreeBSD, $(SYSTEM_UNAME)))
   PLATFORM = linux
   OS_NAME = linux
   OS_VERSION := $(shell uname -r)
